python/py-gmic/Portfile

```diff
@@ -46,10 +37,10 @@ if {${name} ne ${subport}} {
                     port:curl \
                     path:lib/libavcodec.dylib:ffmpeg \
                     port:fftw-3 \
-                    port:jpeg \
+                    port:gmic-lib \
+                    path:include/turbojpeg.h:libjpeg-turbo \
                     port:libpng \
                     port:libomp \
-                    path:lib/opencv4/libopencv_core.dylib:opencv4 \
```

Comment:
Marius, `opencv4` doesn't appear to be in play, so I removed the dependency.

Would `gmic-lib` also need to be built with `opencv4` support enabled, in order to support that here?